### ISSUE TITLE
HTTP 404 Object Not Found

### ISSUE THREAD

**Issue by derks** [boundary/boundary-meter_cookbook#10]
- Ubuntu 14.04 LTS
- Chef Client: 12.1.1

Clean install of latest cookbook from Git, simply added to `Berksfile` and `berks install && berks upload && berks apply`...  I get the following:

```
================================================================================
Recipe Compile Error in /var/chef/cache/cookbooks/boundary-meter/recipes/default.rb
================================================================================

Net::HTTPServerException
------------------------
404 "Object Not Found"

Cookbook Trace:
---------------
  /var/chef/cache/cookbooks/boundary-meter/libraries/boundary_data.rb:35:in `boundary_data_merge'
  /var/chef/cache/cookbooks/boundary-meter/recipes/default.rb:53:in `block in from_file'
  /var/chef/cache/cookbooks/boundary-meter/recipes/default.rb:47:in `from_file'

Relevant File Content:
----------------------
/var/chef/cache/cookbooks/boundary-meter/libraries/boundary_data.rb:

 28:        from_data_bag(value, namespace) ||
 29:        from_node_attr(value, namespace)
 30:      end
 31:
 32:      def boundary_data_merge(namespace='boundary_meter')
 33:        databag, item = namespace.split('_')
 34:
 35>>       return node[namespace] unless data_bag(databag).include?(item)
 36:
 37:        node[namespace].merge(data_bag_item(databag, item).to_hash)
 38:      end
 39:
 40:      private
 41:
 42:      def from_data_bag(value, namespace)
 43:        databag, item = namespace.split('_')
 44:


Running handlers:
Running handlers complete
Chef Client failed. 0 resources updated in 4.690793169 seconds
```

Only Attr set in environment is:

```
"boundary_meter": {
      "token": "api.XXXXXXXXXXXX-XXX"
    },
```

Any help would be appreciated.

**Comment by derks** [comment]
Looks related to:

 * https://github.com/boundary/boundary-meter_cookbook/pull/9


But there is no mention of a required data bag in the README.  Please clarify if possible.  Thanks.

**Comment by derks** [comment]
Merging Pull Request #9 and adding an empty data bag called 'boundary' appears to have resolved the issue.

**Comment by sigil66** [comment]
You should no longer need an empty databag with the merge.

**Comment by sigil66** [comment]
Merged #9
